pkg/terraform/terraform.go

```diff
@@ -93,6 +89,16 @@ func unpackAndInit(dir string, platform string) (err error) {
 		return errors.Wrap(err, "failed to unpack Terraform modules")
 	}
 
+	err = data.Unpack(filepath.Join(dir, executablePath), executablePath)
```

Comment:
This will need `filepath.FroSlas` or whatever on the second arg.

Reply:
TIL. That's a useful function.